custom-properties/set_org_custom_properties.py

```diff
@@ -0,0 +1,131 @@
+import os
+import requests
+import json
+import logging
+
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+
+# GitHub API base URL
+API_BASE = "https://api.github.com"
+
+# Get OAuth token from environment variable
+TOKEN = os.environ.get('OAUTH_TOKEN')
+if not TOKEN:
+    raise ValueError("OAUTH_TOKEN environment variable is not set")
+
+# Your organization name
+ORG_NAME = "hmcts-test"
+
+# Headers for API requests
+headers = {
+    "Authorization": f"Bearer {TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+def define_custom_property(org_name):
+    """
+    Define a custom property for the organization.
```

Comment:
I think have doc strings updated to reflect syntax, example:

`    """
    Prompts the model against each chunk of text.

    Args:
        chunks (list): List of text chunks.
        initial_prompt (list): Initial prompt messages from JSON.
        client (AzureOpenAI): Azure OpenAI client instance.
        deployment_name (str): Deployment name for the model.

    Returns:
        list: List of responses from the model.
    """`